libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/Route.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.base.route.model
+
+import com.mapbox.api.directions.v5.models.RouteLeg
+
+data class Route(
```

Comment:
Same as in https://github.com/mapbox/mapbox-navigation-android/pull/2240#discussion_r344939985

Reply:
Same as in https://github.com/mapbox/mapbox-navigation-android/pull/2240#discussion_r344966253